### DISCUSSION TITLE
Additional File Library Fields

### DISCUSSION THREAD

**Discussion by nathanhere** [directus/directus#6861] [Feature Requests]
I have a use case where, in addition to writing out an image description for accessibility purposes, I would like to add a separate field for information that needs to be displayed as subtext below the image whenever it is displayed (like Photo Credit Name).

**Comment by rijkvanzanten** [comment]
You can go into Settings > Data Model, and enable system collections in the right sidebar. This'll allow you to add custom fields to the system collections 👍🏻
